src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
Use LocalDate instead of Date. Here is some background why you should use it primarily: https://www.baeldung.com/migrating-to-java-8-date-time-api. Replace Date with LocalDate and adjust your code to use the newer API.